topics/microbiome/tutorials/multivariable-association/tutorial.md

```diff
@@ -0,0 +1,213 @@
+>    - *"Interactions: Fixed effects"*: `c3:age`, `c4:diagnosis`
+>    - *"Random effects"*: `c5:subject`
+>    - *"Reference"*: `diagnosis,CD`
+>
+>  Keep rest of the default values as it is.
+{: .hands_on}
+
+# Understanding parameters in the tool
+Lets now understand the role of each parameter in the tool.
+
+1. **Interactions:Fixed effects** : MaAsLin2 doesn't directly support testing for interactions between variables. To explore interactions, users must manually create interaction terms as additional fixed_effects in their analysis.
+
+2. **Random effects** : In some studies, like those following people over time or studying families, samples from the same group can be similar. MaAsLin2 helps handle this by letting researchers choose a grouping factor. This helps make sure the statistical analysis is more accurate.
+
+3. **Reference** : It allows researchers to establish a baseline or standard category against which other categories are compared, helping to interpret and understand the effects of different variables on microbial features. In MaAslin2, reference level is must for variables with more than two distinct kind of values.
+
+   > <comment-title></comment-title>
+   >
+   > - Reference for a variable with more than two levels is provided as a string of `variable,reference`.
+   > - Reference for more than one variable having more than two levels each is provided as a string of `variable1,reference1,variable2,reference2` .
+   > - Example, both diagnosis and site variable have more than two levels hence reference can be provided as `diagnosis,CD,site,Cedars-Sinai`.
+   {: .comment}
+
+4. **Additional options** :
+- min_abundance : The minimum abundance for each feature [ Default: 0 ]
+- min_prevalence:  The minimum percent of samples for which a feature is detected at minimum abundance [ Default: 0.1 ]
+- max_significance: The q-value threshold for significance [ Default: 0.25 ]
+- normalization: The normalization method to apply [ Default: "TSS" ] [ Choices: "TSS", "CLR", "CSS", "NONE", "TMM" ]
+- transform: The transform to apply [ Default: "LOG" ] [ Choices: "LOG", "LOGIT", "AST", "NONE" ]
+- analysis_method : The analysis method to apply [ Default: "LM" ] [ Choices: "LM", "CPLM", "ZICP", "NEGBIN", "ZINB" ]
+- correction : The correction method for computing the q-value [ Default: "BH" ]
+- standardize : Apply z-score so continuous metadata are on the same scale [ Default: TRUE ]
+- plot_heatmap : Generate a heatmap for the significant associations [ Default: TRUE ]
+- heatmap_first_n : In heatmap, plot top N features with significant associations [ Default: 50 ]
+- plot_scatter : Generate scatter plots for the significant associations [ Default: TRUE ]
+- cores : The number of R processes to run in parallel [ Default: 1 ]
+
+
+# Reading Output Files
+The tool generate the following five major files:
+- **Data output files**
+    1. `residuals.rds`
+            This file contains a data frame with residuals for each feature.
```

Comment:
What are the residuals ? Maybe we could add something like this at the end of the tutorial, or at least explain how to use the residuals could be used: https://forum.biobakery.org/t/maaslin2-model-validation/1237/4